package.json

```diff
@@ -175,7 +175,7 @@
         "deepmerge": "^2.1.1",
         "draft-js": "^0.10.1",
         "enzyme": "3.8.0",
-        "enzyme-adapter-react-16": "^1.10.0",
+        "enzyme-adapter-react-16": "^1.12.1",
         "enzyme-to-json": "^3.3.4",
         "eslint": "^4.19.1",
```

Comment:
Has FTL discussed adding the lint plugin for hooks?
https://www.npmjs.com/package/eslint-plugin-react-hooks
https://reactjs.org/docs/hooks-overview.html#rules-of-hooks

Reply:
yes, we will be adding these plugins. I'll discuss it at the next FTL meeting but it'll be an update to @box/frontend